### PR TITLE
fro-37-why-zuri-chat-features: implemented the why zuri chat page usi…

### PR DESCRIPTION
**Pull request by Harbeytee**
…ng css modules mobile and desktop responsive

# Fixes Issue/Linear Ticket
I was tasked with implementing the why zuri chat page Here is the link to the linear ticket.
-it is mobile responsive and implemented to pixel perfect
https://linear.app/mechanic-team-prybar/issue/FRO-37/why-zuri-chat-features

( This could be an existing issue or a linear ticket )

Screenshot (28)

**My PR Closes #issue_number_here** or
**My PR Fixes ID-FRO-37**

# Changes proposed

## What were you told to do?

implement why zuri chat page to pixel perfect

## What did you do?

-the pages were styled using css modules
-no dependencies were installed
-it is mobile and desktop responsive

# Check List (Check all the applicable boxes)

🚨Please review the [style guide for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/STYLING.md) and [guidelines for contributing](https://github.com/zurichat/zc_main/blob/dev/docs/CONTRIBUTING.md) to this repository.

- [x] My code follows the code style of this project.
- [x] This PR does not contain plagiarized content.
- [x] The title and description of the PR is clear and explains the approach.
- [x] I am making a pull request against the **dev branch** (left side).
- [x] My commit messages styles matches our requested structure.
- [x] My code additions will fail neither code linting checks nor unit test.
- [x] I am only making changes to files I was requested to.

# Screenshots/ Videos

Screenshot (28)
Screenshot (30)
Screenshot (31)
Screenshot (32)
Screenshot (33)
Screenshot (34)
Screenshot (35)

### CODE REVIEW

File: packages/main/src/pages/whyzurichat/why.module.css [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    margin-left: auto;
+    margin-right: auto;
+}
+.benefit_boxes h3 {
+    margin-bottom: 1.5rem;
+}
+.benefit_boxes article {
+    width: 100%;
+    
+   /* max-width: 431px;*/
```
Comment (JulianaSau) [inline packages/main/src/pages/whyzurichat/why.module.css:88]: Remove this code if its not being used

File: packages/main/src/pages/whyzurichat/why.module.css [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+    margin-left: 1rem;
+    
+}
+.benefit_boxes {
+    display: grid;
+  grid-template-columns: repeat(auto-fit, minmax(400px, 1fr));
+  
+ width: 100%;
+  
+    /*display: flex;
```
Comment (JulianaSau) [inline packages/main/src/pages/whyzurichat/why.module.css:57]: Remove this code if its not being used

**Review by JulianaSau — CHANGES_REQUESTED** [review]
I'm seeing a why zuri chat page already exists here
https://github.com/zurichat/zc_main/tree/dev/packages/main/src/pages/WhyZuriChat
delete this one and create the whyZuriChat page in this folder. That's where all static pages are. https://github.com/zurichat/zc_main/tree/dev/packages/main/src/pages/static

When you're done include screenshots of the way the page looks now

**Comment by Harbeytee** [comment]
-files have been restructured
-screenshots have been added

**Review by JulianaSau — APPROVED** [review]

**Comment by JulianaSau** [comment]
Why did you close the PR? @Harbeytee

**Comment by Harbeytee** [comment]
sorry it was a mistake i wanted to reopen a new one. thought you weren't going to merge

**Pull request merged**